BoxBrowseSDKSampleApp/Podfile

```diff
@@ -1,6 +1,7 @@
 use_frameworks!
 platform :ios, '8.0'
 
-target 'BoxBrowseSDKSampleApp' do
+target "BoxBrowseSDKSampleApp" do
+	pod 'box-ios-sdk', :git => "https://github.com/box/box-ios-sdk.git", :branch => "nsurlsession"
```

Comment:
Are we going to push this as-is and then take it out when we merge the nsurlsession branch to master?